internal/provider/kratos/kratos.go

```diff
@@ -16,16 +18,19 @@ import (
 type IdentityPublicMetadata struct {
 	// Indicate identity is created with auto generated password.
 	ForceReset bool
+	// Associated paralus partner and organization
+	Organization string
+	Partner      string
 }
 
 type kratosAuthProvider struct {
 	kc *kclient.APIClient
 }
 type AuthProvider interface {
 	// create new user
-	Create(context.Context, string, map[string]interface{}, bool) (string, error) // returns id,error
+	Create(context.Context, string, map[string]interface{}, map[string]interface{}) (string, error) // returns id,error
```

Comment:
Can we use `IdentityPublicMetadata` instead of map as parameter for Create and Update function?
I think this will make use Organization/Partner/ForceReset directly without need to converting interface{} to their types.

Reply:
@akshay196 this is taken care of